sql-cli/README.md

```diff
@@ -11,7 +11,7 @@ The SQL CLI component in OpenSearch is a stand-alone Python application and can
 
 It only supports [OpenSearch SQL Plugin](https://docs-beta.opensearch.org/search-plugins/sql/index/)
 You must have the OpenSearch SQL plugin installed to your OpenSearch instance to connect. 
-Users can run this CLI from MacOS and Linux, and connect to any valid OpenSearch end-point such as Amazon Elasticsearch Service (AES).
+Users can run this CLI from Unix/Linux like OS or Windows, and connect to any valid OpenSearch end-point such as Amazon Elasticsearch Service (AES).
```

Comment:
i think `/Linux` is redundant since "unix like" already includes Linux, and also change "Amazon Elasticsearch Service" to "Amazon OpenSearch Service"?

Reply:
good catch, will do